client/src/Pages/RecruiterAddEditBoothPage.js

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import { Tab, Tabs, Card, Button, InputGroup, FormControl} from "react-bootstrap";
+import profile from '../Images/profile.jpg'; 
+// import profile from '../Images/profile.jpg'; 
+
+export default class RecruiterAddEditBoothPage extends React.Component {
+    handleRoute = route => () => {
+        this.props.history.push({ pathname: route });
+        };
+  render() {
+    return (
+        <div style={{"padding": "20px"}}>
+            <Card style={{"box-shadow": "8px 4px 8px 4px rgba(0,0,0,0.2)", "padding": "20px"}}>
+                <Tabs defaultActiveKey="add" id="uncontrolled-tab-example">
+                <Tab eventKey="add" title="Add New Booth">
+                    <br></br>
+                    <h2>Add New Booth at Harvey Mudd College Computer Science Fair</h2>
+                    <Card style={{"margin-top": "20px"}}>
+                        <div style={{"padding": "20px"}}>
+                            <InputGroup className="mb-3">
+                                <InputGroup.Prepend>
+                                <InputGroup.Text id="basic-addon1">Company: </InputGroup.Text>
+                                </InputGroup.Prepend>
+                                <FormControl
+                                placeholder="Enter company name"
+                                aria-label="Enter company name"
+                                aria-describedby="basic-addon1"
+                                />
```

Comment:
Since recruiters are tied to a company, I don't think they should see an option to enter company name. That will be 'autofilled' due to their company affiliation.

Reply:
Oh yeah huh ahaha will do!